src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution_ArgsToParameters.cs

```diff
@@ -164,6 +174,44 @@ public ImmutableArray<int> ToImmutableArray()
                 ArgumentAnalysisResult.NormalForm(argsToParameters.ToImmutableArray());
         }
 
+        private static int? CheckForBadNonTrailingNamedArgument(AnalyzedArguments arguments, ParameterMap argsToParameters, ImmutableArray<ParameterSymbol> parameters)
+        {
+            // Is there any named argument used out-of-position and followed by unnamed arguments?
+
+            // If the map is trivial then clearly not.
+            if (argsToParameters.IsTrivial)
+            {
+                return null;
+            }
+
+            // Find the first named argument which is used out-of-position or a params parameter
+            int foundPosition = -1;
+            int length = argsToParameters.Length;
```

Comment:
>argsToParameters [](start = 25, length = 16)

I think it will be clearer if ```arguments.Length``` was used instead of ```argsToParameters.Length```.